website/src/components/Messages/MessageConversation.tsx

```diff
@@ -9,16 +10,21 @@ interface MessageConversationProps {
 }
 
 export function MessageConversation({ messages, enableLink, highlightLastMessage }: MessageConversationProps) {
+  const { t } = useTranslation("message");
   return (
     <Stack spacing="4">
-      {messages.map((message, idx) => (
-        <MessageTableEntry
-          enabled={enableLink}
-          message={message}
-          key={message.id + message.frontend_message_id}
-          highlight={highlightLastMessage && idx === messages.length - 1}
-        />
-      ))}
+      {!messages || messages.length === 0 ? (
```

Comment:
```suggestion
      {messages.length === 0 ? (
```
No need to check messages is fasly, because the type already expressed that

Reply:
Perfect, I'll change that 👍